ceremony/2022-09-27/staged/targets.json

```diff
@@ -0,0 +1,203 @@
+{
+	"signed": {
+		"_type": "targets",
+		"spec_version": "1.0",
+		"version": 5,
+		"expires": "2022-12-27T19:57:02Z",
+		"targets": {
+			"artifact.pub": {
+				"length": 177,
+				"hashes": {
+					"sha256": "59ebf97a9850aecec4bc39c1f5c1dc46e6490a6b5fd2a6cacdcac0c3a6fc4cbf",
+					"sha512": "308fd1d1d95d7f80aa33b837795251cc3e886792982275e062409e13e4e236ffc34d676682aa96fdc751414de99c864bf132dde71581fa651c6343905e3bf988"
+				},
+				"custom": {
+					"sigstore": {
+						"status": "Active",
+						"usage": "Unknown"
+					}
+				}
+			},
+			"ctfe.pub": {
+				"length": 177,
+				"hashes": {
+					"sha256": "7fcb94a5d0ed541260473b990b99a6c39864c1fb16f3f3e594a5a3cebbfe138a",
+					"sha512": "4b20747d1afe2544238ad38cc0cc3010921b177d60ac743767e0ef675b915489bd01a36606c0ff83c06448622d7160f0d866c83d20f0c0f44653dcc3f9aa0bd4"
+				},
+				"custom": {
+					"sigstore": {
+						"status": "Active",
+						"uri": "https://ctfe.sigstore.dev",
+						"usage": "CTFE"
+					}
+				}
+			},
+			"ctfe/ctfe.pub": {
+				"length": 177,
+				"hashes": {
+					"sha256": "7fcb94a5d0ed541260473b990b99a6c39864c1fb16f3f3e594a5a3cebbfe138a",
+					"sha512": "4b20747d1afe2544238ad38cc0cc3010921b177d60ac743767e0ef675b915489bd01a36606c0ff83c06448622d7160f0d866c83d20f0c0f44653dcc3f9aa0bd4"
+				},
+				"custom": {
+					"sigstore": {
+						"status": "Active",
+						"uri": "https://ctfe.sigstore.dev",
+						"usage": "CTFE"
+					}
+				}
+			},
+			"ctfe/ctlog2022.pub": {
+				"length": 178,
+				"hashes": {
+					"sha256": "bf786e715151f2544fa7a6faadd9ddb7955c81015be9f706d3de82de157f06b4",
+					"sha512": "5e8dcb6eec4ffedd1c4825b6a8f9db8b5ba2db21224db74250bd80ec0947882e6b07e9a708f076cbc63801efd70d7c818bafde367026229d0deae7b7d7edaa43"
+				},
+				"custom": {
+					"sigstore": {
+						"status": "Active",
+						"uri": "https://ctfe.sigstore.dev",
+						"usage": "CTFE"
+					}
+				}
+			},
+			"fulcio.crt.pem": {
```

Comment:
To confirm, this is correct, that we want the targets duplicated under top level and nested under a folder?

Reply:
I don't want to immediately break sigstore-rs: https://github.com/sigstore/sigstore-rs/issues/134

but with the updated paths we can allow them to migrate and deprecate the old regexes